README.md

```diff
@@ -1,19 +1,12 @@
 # Kani Rust Verifier
 
-The Kani Rust Verifier aims to be a bit-precise model-checker for Rust.
-Kani ensures that unsafe Rust code is actually safe, and verifies that safe Rust code will not panic at runtime.
+The Kani Rust Verifier aims to be a bit-precise model checker for Rust.
 
-## Installing Kani
+Kani verifies memory safety, checks for different variants of undefined behavior, panics and user-defined assertions - even in unsafe Rust code.
```

Comment:
A few suggestions:
- I think we should tie "unsafe Rust code" to memory safety because people assume that memory safety is checked by the compiler.
- Move checking of undefined behavior to the end.
Here is a suggested re-wording:
```suggestion
Kani verifies memory safety -- even in unsafe Rust code, verifies user-specified assertions as well as the absence of panics, and checks for some classes of undefined behavior.
```

Reply:
I liked your re-wording but at the same time thought it may be confusing. I made an attempt to convert this sentence into a bulleted list. How does it look to you?